app/org/sagebionetworks/bridge/dynamodb/DynamoSurveyInfoScreen.java

```diff
@@ -29,7 +29,8 @@ public DynamoSurveyInfoScreen(SurveyElement entry) {
         setIdentifier(entry.getIdentifier());
         setGuid(entry.getGuid());
         setData(entry.getData());
-        setRules(entry.getRules());
+        setBeforeRules(entry.getBeforeRules());
+        setAfterRules(entry.getAfterRules());
```

Comment:
Do we have a unit test to make sure these rules are being properly pulled in from the SurveyElement?